### DISCUSSION TITLE
Exploring the Future of Programming

### DISCUSSION THREAD

**Discussion by TobestDev** [community/community#123482] [Other Feature Feedback, Questions, & Ideas]
Topic: Future of programming

Question
What are your thoughts on the future of programming? Let's discuss.
### Body

As technology propels forward, the future of programming seems boundless. In this discussion, we invite developers to share their insights, predictions, and concerns regarding the trajectory of programming. From the rise of AI to the impact of quantum computing, what do you envision for the future of programming? Join us as we explore the evolving landscape of programming and its implications for developers worldwide. Your contributions will help shape the conversation and illuminate the path forward in this ever-changing field. 
What are your thoughts on the future of programming? Let's discuss.

**Comment by CiceroLino** [comment]
Maybe, after many years (hundred or thousand years) programming computers, next target will be our DNA. Programming our genetics and kind of stuff. 

Reasons, avoid illnesses, dna malfunction(cancer), get more adaptily to enviroment and more

And we cant forget, the best computer still are our brains

**Reply by CopperEagle** [reply]
Good point. There is something called [cell reprogramming](https://en.wikipedia.org/wiki/Reprogramming), used for therapies and rejuvenation research. Has little to do with what we programmers do though :laughing:

**Comment by CopperEagle** [comment]
:thinking:  Programming is a way of specifying behavior. There are many ways to do this, different languages. Each language has different tradeoffs. Assembly is easy for a machine to understand. Mathmatics is as powerful a description we can deliver while being fully precise. And now, LLMs may allow us to use plain language to specify behavior to the machine (having them write code given a description). Confusion assured.

**Comment by ebndev** [comment]
You all might be interested in joining the [#opensourcefriday AI and OSS](https://github.com/orgs/community/discussions/123787) chat happening this Friday, May 17th. 😄 Please feel free to leave your thoughts and questions in that discussion as well!
